ProfessorSpace/src/controller/Controller.java

```diff
@@ -0,0 +1,36 @@
+package controller;
+
+import java.util.ArrayList;
+
+import model.Question;
+import view.InputView;
+
```

Comment:
Add javadocs:
1. Each file should specify the what the class does.
2. Add the authour and version for each file.
3. Go through the code pushed by @sarthak-tiwari for reference.

Reply:
Yes, forget about that will do so of the same style.